components/decimal/benches/fixed_decimal_format.rs

```diff
@@ -26,12 +26,14 @@ fn triangular_nums(range: f64) -> Vec<isize> {
 
 fn overview_bench(c: &mut Criterion) {
     let nums = triangular_nums(1e9);
+    let provider = icu_provider_adapters::any_payload::AnyPayloadProvider::new_default::<
+        icu_decimal::provider::DecimalSymbolsV1Marker,
+    >();
```

Comment:
`question`: is that the best we can do? This looks... awkward. I know it's a benchmark, but code like this is what people look into to copy&paste for samples. I'd like to keep benches and examples relatively clean. Is there no abstraction that would make it cleaner? (it's ok if there isn't, just flagging in hopes there's one :))

Reply:
What do you think it's awkward about this? I could add imports, so this becomes

```
AnyPayloadProvider::new_default::<DecimalSymbolsV1Marker>()
```

Do you still think that's awkward?